app/src/main/java/com/android/streetworkapp/ui/train/TrainHubScreen.kt

```diff
@@ -105,7 +105,24 @@ fun TrainHubScreen(
                               "Muscle-up" to false)
                           .first { it.first == activity }
                 },
-                getItemImageRes = { R.drawable.pushup },
+                getItemImageRes = { role ->
+                  when (role) {
+                    // source of icone :
+                    // https://www.shutterstock.com/image-vector/bundle-set-sport-pictogram-gym-fitness-2519719005
+                    // https://www.amazon.in/T%C3%BCnde-K%C3%A1tai-Burpee-Challenge/dp/B07HRTGM1K
+                    // edited in GIMP
+                    "Push-ups" -> R.drawable.train_pushup
```

Comment:
You might want to put the link to (ressources / apps used) inside your PR instead of in the code. What do you think ?

Reply:
i'll do both